Hoxtygen/Library-react
# Chore add coveralls code coverage backend #169110905

# What does this PR do
Adds coveralls to track test coverage

## Description of task to be completed
- Add coveralls package to the project to track the test coverage
- Add coveralls badge to readme

## How should this manually be tested
- clone the repo
- open the terminal and `cd` in to the project folder
- run `npm install`
- run `npm test` to see test and test coverage

## What are the relevant pivotal tracker stories
[Delivers [#169110905](https://www.pivotaltracker.com/story/show/169110905)]

## Screenshots (If appropriate)
<img width="616" alt="Screenshot 2019-11-17 at 09 24 48" src="https://user-images.githubusercontent.com/19408669/69005040-2b651300-091c-11ea-8a84-732fa3718cdc.png">



## Questions (If any)
